src/main/java/pl/agh/cs/io/WindowsPerExe.java

```diff
@@ -0,0 +1,43 @@
+package pl.agh.cs.io;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class WindowsPerExe {
```

Comment:
nazwa sugeruje, że ta klasa przechowuje wiele okien dla jednego `exe`, a z tego co rozumiem to tak nie jest. Może lepiej `ProcessesPerExe` lub `ExeProcesses`?

Reply:
ok